### PR TITLE
Added EiskaltDC++.app v2.3.0

### PR DESCRIPTION
**Pull request by torrottum**

### CODE REVIEW

File: Casks/eiskaltdcpp.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'eiskaltdcpp' do
+  version '2.3.0'
+  sha256 '86bcc03c00758efc507030c4a7a16c6d04b9ae763b75255dadbc1b4144a7b34f'
+
+  url 'http://downloads.sourceforge.net/sourceforge/eiskaltdcpp/EiskaltDC++-2.3.0-8c41579-x86_64-MacOSX10.9.dmg'
+  name 'EiskaltDC++'
+  homepage 'https://code.google.com/p/eiskaltdc/'
+  license :gpl    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (fanquake) [inline Casks/eiskaltdcpp.rb]: Given that you've specified a license, can you remove the #todo comment

**Comment by fanquake** [comment]
Can you now just squash your commits?

**Comment by torrottum** [comment]
@fanquake Done!

File: Casks/eiskaltdcpp.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'eiskaltdcpp' do
+  version '2.3.0'
+  sha256 '86bcc03c00758efc507030c4a7a16c6d04b9ae763b75255dadbc1b4144a7b34f'
+
+  url 'http://downloads.sourceforge.net/sourceforge/eiskaltdcpp/EiskaltDC++-2.3.0-8c41579-x86_64-MacOSX10.9.dmg'
+  name 'EiskaltDC++'
+  homepage 'https://code.google.com/p/eiskaltdc/'
```
Comment (vitorgalvao) [inline Casks/eiskaltdcpp.rb]: Google code is going away, so an alternative homepage should be found, like [sourceforge](http://sourceforge.net/projects/eiskaltdcpp/).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It will be merged in the pull request referenced above due to some simple recommended alterations. Your contribution is still included (and still credited to you), with the appropriate modifications, when applicable.

**Pull request closed without merging**